open_spiel/python/algorithms/efr_test.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 DeepMind Technologies Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for open_spiel.python.algorithms.efr."""
+
+from absl.testing import absltest
+from absl.testing import parameterized
+import numpy as np
+
+from open_spiel.python import policy
+from open_spiel.python.algorithms import efr
+from open_spiel.python.algorithms import expected_game_score
+import pyspiel
+
+_KUHN_GAME = pyspiel.load_game("kuhn_poker")
```

Comment:
Don't make these global. You can put them into the object, by initializing them in a setup function in the EFRTest class. The signature is:

```
def setUp(self):
```